config-linux.md

```diff
@@ -509,10 +508,8 @@ The actions, architectures, and operators are strings that match the definitions
 The following parameters can be specified to set up seccomp:
 
 * **`defaultAction`** *(string, REQUIRED)* - the default action for seccomp. Allowed values are the same as `syscalls[].action`.
-
 * **`architectures`** *(array of strings, OPTIONAL)* - the architecture used for system calls.
     A valid list of constants as of libseccomp v2.3.2 is shown below.
-
```

Comment:
Why are you removing this line?  A blank line before and after a Markdown list is traditional, and [it renders fine][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/75c847a4e6f4f2e5d13a2477fd57d5ceb98252dc/config-linux.md#seccomp

Reply:
I removed them, because the blank line does not make sense.
But if it's traditional, I will keep them.